docker/Dockerfile

```diff
@@ -1,16 +1,21 @@
 FROM python:3.10
```

Comment:
```suggestion
FROM python:3.10
LABEL org.opencontainers.image.source=https://github.com/NeurodataWithoutBorders/nwbwidgets
```

The GHCR likes having these set

Reply:
@CodyCBakerPhD can you send me some reference for this? I'm interested in why that is the case, might use it in other places as well if it's important

Reply:
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images

Reply:
description and licenses also not a terrible idea I suppose but I haven't messed with that yet

Reply:
but setting these helped solve any problem, in specific? I still can't see a clear advantage of doing that
usually we can tag the same image with multiple targets, e.g. if we want to push to ECR, Dockerhub, Azure CR, etc... and in some cases that might be interesting to do. Why would we want to fix that label to ghcr?

Reply:
![image](https://github.com/NeurodataWithoutBorders/nwbwidgets/assets/51133164/04bbf3c0-cc36-4a9d-8a42-6208ac18387d)

When I did it to my images, everything worked as expected. I believe I recall seeing errors otherwise